primitives/Cargo.toml

```diff
@@ -6,13 +6,20 @@ edition = "2021"
 
 [dependencies]
 bounded-collections = { version = "0.1", default-features = false }
-scale = { package = "parity-scale-codec", version = "3", default-features = false, features = ["derive"] }
+
+scale = { package = "parity-scale-codec", version = "3.6.9", default-features = false, features = ["derive"] }
+scale-decode = { version = "0.10.0", default-features = false, features = ["derive"], optional = true }
```

Comment:
Can these not be added as features on the scale dependency?

Reply:
Reference taken from [1](https://github.com/use-ink/ink/blob/0cd2b897586194afca08714216f1ed876d05754d/Cargo.toml#L60) & [2](https://github.com/use-ink/ink/blob/0cd2b897586194afca08714216f1ed876d05754d/crates/primitives/Cargo.toml#L21)